### PR TITLE
Add reference to the published article.

### PR DESCRIPTION
**Pull request by mpoullet**

### CODE REVIEW

**Comment by mpoullet** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # RED: version 1.0.0
 
 Demonstration of the image restoration experiments conducted in 
-Y. Romano, M. Elad, and P. Milanfar, "The Little Engine that Could: 
-Regularization by Denoising (RED)", submitted to SIAM Journal on 
-Imaging Sciences, 2016. https://arxiv.org/abs/1611.02862.
+Y. Romano, M. Elad, and P. Milanfar, "[The Little Engine that Could: Regularization by Denoising (RED)](https://epubs.siam.org/doi/10.1137/16M1102884)", SIAM Journal on Imaging Sciences, 10(4), 1804–1844, 2017 [[arXiv](https://arxiv.org/abs/1611.02862)].
```
Comment (sungjoonchoi) [inline README.md]: Please break the line.

**Review by sungjoonchoi — CHANGES_REQUESTED** [review]

**Comment by mpoullet** [comment]
Line breaked.

**Review by sungjoonchoi — APPROVED** [review]

**Comment by sungjoonchoi** [comment]
Looks good! Thanks for your changes.

**Pull request merged**